.github/workflows/test.yml

```diff
@@ -24,8 +24,7 @@ jobs:
       - name: Install dependencies
         run: |
           npm config set '//registry.npmjs.org/:_authToken' "${{ secrets.NPM_AUTH_TOKEN }}" \
-          && yarn install --frozen-lockfile
+          && yarn install --frozen-lockfile && yarn build
```

Comment:
I think having `yarn build` here would've prevented [this PR](https://github.com/Uniswap/unified-routing-api/pull/206) that broke the pipeline from getting in